supabase/migrations/20250430195107_contacts.sql

```diff
@@ -0,0 +1,130 @@
+CREATE OR REPLACE FUNCTION wallet.find_user_profiles_by_partial_username(partial_username text, current_user_id uuid)
+RETURNS TABLE (username text, id uuid) 
+LANGUAGE plpgsql
+SECURITY DEFINER
+AS $$
+BEGIN
+  IF length(partial_username) < 3 THEN
+    RETURN;
```

Comment:
is this return value aligned with the defined returned type of the db? have you tried executing this function with value shorter then 3 to check if it works?